problem-sets/2019-10-30/2019-10-30.md

```diff
@@ -0,0 +1,169 @@
+## AutoCompete 2019-10-30
+
+### Mummies in a Pyramid
+
+Write a program called `pyramid` that reads a single integer from standard input  
+then print out a pyramid (using asterisks) that encloses mummies (using $ symbol)  
+Mummies should never be outside or on the edge of the pyramid.  
+Tests will run against integers from 1-100 inclusive
+
+Sample Input: `1`  
+Expected Output :
+```
+*
+```
+
+Sample Input: `2`  
+Expected Output :
+```
+ *
+***
+```
+
+
+Sample Input: `3`  
+Expected Output :
+```
+  *
+ *$*
+*****
+```
+
+
+Sample Input: `4`  
+Expected Output :
+```
+   *
+  *$*
+ *$$$*
+*******
+```
+
+Sample Input: `5`  
+Expected Output :
+```
+    *
+   *$*
+  *$$$*
+ *$$$$$*
+*********
+```
+
+### Haunted Elevator
+
+The Haunted Elevator belongs to a building that has no basement and will always start on floor 1.  
+The first elevator press will move the elevator in the desired direction by 1 floor.  
+If the elevator is on floor 1 and wants to go DOWN one step, the elevator is still on floor 1.  
+The Haunted Elevator will reverse the direction after every other button press.
```

Comment:
It wasn't clear to me at first what it means that the elevator reverses direction on every other button press. Can we add this explanation to make that process more clear?

If the elevator never got haunted here is how it would react to someone pressing `UP DOWN UP`
* The elevator starts at floor 1
* Pressing UP moves the elevator from floor 1 to floor 2
* Pressing DOWN moves elevator from floor 2 back to floor 1
* Pressing UP moves the elevator from floor 1 to floor 2

But, that's not how things go. This elevator gets haunted every other the button is pressed and the elevator goes in the opposite direction of the button that's pressed.

* The elevator starts on floor 1 and is not haunted.
* Pressing UP causes the elevator to go from floor 1 to 2 it is now haunted.
* Pressing DOWN causes the haunted elevator to go from floor 2 to 3 and now it is not haunted.
* Pressing UP causes the un-haunted elevator to go from floor 3 to floor 4.
* The elevator went from floor 1 to floor 4 for a total of 3 floors.

Reply:
thanks!